src/OrbitQt/FilterPanelWidget.h

```diff
@@ -8,8 +8,6 @@
 #include <QFrame>
 #include <QWidget>
 
-#include "ui_FilterPanelWidget.h"
-
 namespace Ui {
 class FilterPanelWidget;
 }
```

Comment:
Just to clarify. The best practice here is to include `ui_FilterPanelWidget.h` in FilterPanelWidget.cpp and in FilterPanelWidget.h this forward declaration (i hope this is the right term) aka
```
namespace Ui {
class FilterPanelWidget;
}
```
?

Reply:
Yes, exactly. (Forward declaration is the right term)